### ISSUE TITLE
revisiting the Language Table....

### ISSUE THREAD

**Issue by retabak** [bootstrapworld/curriculum#1573]
As I was looking at the lesson table to give feedback, I remembered the existance of the Language Table. 

I know we've chatted about it before - and maybe even added a pedagogy box? (I can't find it!) - but as I clicked on it today, I agin found myself wondering how and why it might be useful, even though I am *totally* confident you've told me before. 

If I *work* at Bootstrap and I can't articulate how / why it's useful, I suspect it perplexes teachers and other folks who visit our website.

I propose we say goodbye to it and make it a thing that teachers learn about in PD. (Do they learn about it in PD?! I don't remember learning about it when I attended PD.)

If others are attached to it (@schanzer @flannery-denny ??), I think we need to add a facilitator note in the lesson table in every lesson which it appears, which feels excessive to me.

**Comment by flannery-denny** [comment]
@schanzer has told me that some teachers display this during class. We have moved it from being part of the lesson plan to being a link in the materials section since I've joined the team. I am definitely not attached to it, nor do I believe that I would teach with it if I were back in the classroom. That said, in the weeks of the year that I was teaching with Bootstrap materials, I did always have a printed version of the contracts page taped to my board for students to refer to if they couldn't find theirs.
